test/python/tensorflow/tf_unittest_runner.py

```diff
@@ -198,37 +209,51 @@ def run_test(test_list, verbosity=2):
     of every test and the result.
     """
     loader = unittest.TestLoader()
-    result = []
     succeeded = []
     failures = []
     errors = []
     for test in test_list:
         tests = loader.loadTestsFromName(test)
         test_result = unittest.TextTestRunner(verbosity=verbosity).run(tests)
-        result.append([test, test_result])
         if test_result.wasSuccessful():
             succeeded.append(test)
-            test_result.testsRun += 1
         elif test_result.failures:
             failures.append(test)
-            test_result.testsRun -= 1
         elif test_result.errors:
             errors.append(test)
+
+    print('\033[1m' + '\n==SUMMARY==' + '\033[0m')
     for test_name in succeeded:
-        print(test_name + ' ..PASS')
+        print(test_name + '\033[92m' + ' ..PASS' + '\033[0m')
     for test_name in failures:
-        print(test_name + ' ..FAIL')
+        print(test_name + '\033[91m' + ' ..FAIL' + '\033[0m')
     for test_name in errors:
-        print(test_name + ' ..ERROR')
+        print(test_name + '\033[93m' + ' ..ERROR' + '\033[0m')
+    return ([succeeded, 1], [failures, -1])
```

Comment:
Lets just return `[succeeded, failures]`. Then we know element 0 is success tests cases, element 1 is fail cases.
You could also return a dictionary, whose string keys `PASSED`, `FAILED` would provide more context than expecting the reader to know elem 0 is success, elem 1 is failures